### DISCUSSION TITLE
Returns back to the main menu, requesting you to login - after logging in

### DISCUSSION THREAD

**Discussion by SpooRe91** [Mijago/D2ArmorPicker#91] [General]
Returns back to the main menu, requesting you to login - after logging in. I believe that the OAuth is not fully in sync, and it redirects me directly back to the main menu 
![image](https://user-images.githubusercontent.com/85784810/210009379-02513ab1-9005-42b0-a708-a77cb6968441.png)

**Comment by TheMojoDojo** [comment]
Hey, the Bungie API is currently down, so nothing will be working. You can check @BungieHelp for updates

**Reply by SpooRe91** [reply]
Thanks my man, love your app!

**Reply by SpooRe91** [reply]
Also, may I suggest some kind of global notification, which should pop-out every time the API is down or servers are down for example, so users are aware of the current situation?

**Reply by TheMojoDojo** [reply]
That is planned for the future, expecting to rework the UI, and such a thing will likely be implemented then.

**Comment by TheYeshir13** [comment]
This discussion will be resolved with https://github.com/Mijago/D2ArmorPicker/issues/105
